<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 220, 220);</string>
  </property>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PreviewButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopPreviewButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>StopPreview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>461</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="RoadLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>531</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/Road.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="offlinedata">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>offlineData</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savedata">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SaveData</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopsave">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>StopSave</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetupAlarm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SetupAlarm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseAlarm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CloseAlarm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
